spec/features/users/user_can_delete_their_own_comments_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+describe 'when a user visits a question show page' do
+  it 'they can delete question comments owned by them' do
+    user = Fabricate(:user, id: 1)
+    question = Fabricate(:question, user: user)
+    comment = question.comments.create!(body: "Nice comment", user_id: user.id)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
```

Comment:
Great testing suite. Really happy with the specificity and different path checking.